lib/test-support/test-index.js

```diff
@@ -25,7 +22,7 @@ if (process.platform === 'win32') {
 
 let emberAppDir = resolve(dirname(indexPath), '..');
 let relPath = relative(emberAppDir, indexPath);
-const emberAppLocation = `serve://dist/${relPath}${indexQuery}`;
+let emberAppLocation = `serve://dist/${relPath}${indexQuery}`;
```

Comment:
Was this `const` changed to `let` just to match its neighbors? (Doesn't look like it get changed)

Reply:
Heh, don't really want to get into the `const` vs. `let` debate, although I think this case is particularly ambiguous. I'd prefer to leave it as-is, if you'd like to disagree and discuss this particular case, I'm happy to, and if you'd like to have the stylistic discussion of `const` vs. `let` in general, hit me up on Discord 😄

Reply:
No, no, it's fine. I wasn't trying to get into a debate. I was just a bit confused as to why it was changed. The commit said "get tests working" but this didn't appear to be needed to get tests working.

Reply:
Yeah, fair, there was so much code change here that I tried to rebase things to make the commits sensible, but didn't go so far as to do the pain of resolving conflicts and splitting out changes I accidentally combined.